apps/eclipse/src/components/chart-examples/README.md

```
@@ -0,0 +1,88 @@
+<BasicBarChartExample />
+```
+
+Or import from the index file:
+
+```tsx
+import { BasicBarChartExample, LineChartExample } from "@/components/chart-examples";
+```
+
+## File Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)